### PR TITLE
 SOL-217 Adding analytics tracker for search request

### PR DESCRIPTION
**Pull request by dino-cikatic**
@martynjames please take a look.

### CODE REVIEW

File: search/views.py [hunk 1]
```diff
@@ -64,7 +67,18 @@ def do_search(request, course_id=None):
                 raise InvalidPageSize(_('Invalid page size of {page_size}').format(page_size=size))
 
             if "page_index" in request.POST:
-                from_ = int(request.POST["page_index"]) * size
+                page = int(request.POST["page_index"])
+                from_ = page * size
+
+        #Google Analytics - log search request
```
Comment (mulby) [inline search/views.py]: Note this call is emtting events just to the tracking logs. You have to make a separate call to emit events to Segment.io (which, in turn, passes them on to google analytics).

Comment (martynjames) [inline search/views.py]: Thanks @mulby - for now our requirement is to emit events to the tracking logs - the comment herein is incorrect. Sorry for any confusion.

**Comment by martynjames** [comment]
@dino-cikatic - can you fix the failures here

**Comment by mulby** [comment]
Is this being deployed separately from the LMS?

**Comment by mulby** [comment]
@lamagnifica

**Comment by dino-cikatic** [comment]
Closing and replacing with: https://github.com/edx/edx-search/pull/5

**Pull request closed without merging**